apps/condo/domains/ticket/utils/helpers.ts

```diff
@@ -649,4 +649,35 @@ export function hasUnreadResidentComments (lastResidentCommentAt, readResidentCo
     }
 
     return false
+}
+
+/** analyticsData array validation function. 
+ * The array can only be empty on the first render. Even if there are no tickets, mock objects are placed in analyticsData. 
+ * For the correct operation of the unload button in Excel, it is necessary to check the objects nested in the array for the presence of at least one ticket 
+ * analyticsData array example: 
+ * [0:{
+        assignee: null
+        categoryClassifier: null
+        count: 0
+        dayGroup: "23.05.2022"
+        executor: null
+        property: null
+        status: "Открыта"
+        __typename: "TicketGroupedCounter"
+        }
+    ]
+*/
+export function isEmptyAnalyticsData (analyticsData) {
+    if (analyticsData === null) {
```

Comment:
```
if (isNull(analyticsData) || isUndefined(analyticsData) || isEmpty(analyticsData)) return true

// no need to use Array.some because you don't return anything so it will iterate up to the end of the list
const count = analyticsData.reduce((count, date) => count + get(date, 'count'), 0)

return count < 1
```